pymc3/distributions/dist_math.py

```diff
@@ -365,6 +365,7 @@ def conjugate_solve_triangular(outer, inner):
             grad = tt.triu(s + s.T) - tt.diag(tt.diagonal(s))
         return [tt.switch(ok, grad, floatX(np.nan))]
 
+
 class SplineWrapper (theano.Op):
```

Comment:
[pep8](https://www.python.org/dev/peps/pep-0008/#id27) does not recommend spacing before opening brackets